crypto_sign/sphincs-haraka-128f-robust/clean/hash_haraka.c

```diff
@@ -14,6 +14,11 @@ void PQCLEAN_SPHINCSHARAKA128FROBUST_CLEAN_initialize_hash_function(
     PQCLEAN_SPHINCSHARAKA128FROBUST_CLEAN_tweak_constants(hash_state_seeded, pub_seed, sk_seed, SPX_N);
 }
 
+void PQCLEAN_SPHINCSHARAKA128FROBUST_CLEAN_destroy_hash_function(
+    hash_state *state) {
```

Comment:
This will need a comment to instruct the linter to ignore that `*state` could be const. See `initialize_hash_function` for SHAKE256.